public/index.js

```diff
@@ -34,24 +34,39 @@ module.exports = function(currentUser, matchingUsers, params, token) {
       window.location = domain + 'authorize?' + query;
     };
 
+    var authorizeSaml = function(domain, qs) {
+      let clientId = qs["client_id"];
+      let connection = encodeURIComponent(qs["connection"]);
+      let link_account_token = encodeURIComponent(qs["link_account_token"]);
+      window.location = domain + `samlp/${clientId}?connection=${connection}&link_account_token=${link_account_token}&prevent_sign_up=true`;
```

Comment:
Should we send the original RelayState here as well?

Reply:
Thanks! Fixed.
![image](https://user-images.githubusercontent.com/331835/141792295-18c9c7a1-7611-4b6f-b749-24ca3c2a7fdc.png)